src/Workleap.OpenApi.MSBuild/OasdiffManager.cs

```diff
@@ -5,7 +5,7 @@ namespace Workleap.OpenApi.MSBuild;
 
 internal sealed class OasdiffManager : IOasdiffManager
 {
-    private const string OasdiffVersion = "1.9.6";
+    private const string OasdiffVersion = "1.10.11";
```

Comment:
should this be hardcoded or renovated?

Reply:
Both. We download the tools so we need a specific version to target but we could add a custom renovate parser to monitor oasdiff and spectral.